hazelcast/src/main/java/com/hazelcast/map/impl/proxy/MapProxySupport.java

```diff
@@ -638,6 +691,12 @@ protected void getAllObjectInternal(List<Data> keys, List resultingKeyValuePairs
                     resultingKeyValuePairs.add(toObject(value));
                 }
             }
+
+            int size = resultingKeyValuePairs.size();
+            if (size > 0 && statisticsEnabled) {
+                localMapStats.incrementHits(size);
```

Comment:
shouldn't the hit be increased half of the size? `resultingKeyValuePairs ` is holding all key-value pairs.